cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2287,7 +2287,7 @@ search = `cs_o365` (sourcetype="ms:o365:reporting:messagetrace" OR sourcetype="o
 | lookup cs_email_sent_upperbound.csv SenderAddress OUTPUT upperBound, avg, stdev \
 | where count>upperBound AND count > `cs_email_increase_over_baseline_limit` \
 | eval avg=round(avg, 2), stdev=round(stdev, 2), upperBound=round(upperBound, 2) \
-| eval cyences_severity = case(count>3*upperBound, "critical", count>2*upperBound, "high", true(), "medium") \
+| eval cyences_severity = case(count>10*upperBound, "critical", count>5*upperBound, "high", true(), "medium") \
```

Comment:
@VatsalJagani 10 is too high. I think 6 would be idea. Checkout six sigma rule/analysis

Reply:
```suggestion
| eval cyences_severity = case(count>6*upperBound, "critical", count>4*upperBound, "high", true(), "medium") \
```